### ISSUE TITLE
How to use encoding KSC_5601?

### ISSUE THREAD

**Issue by oz-k** [hgourvest/node-firebird#328]
I checked issue https://github.com/hgourvest/node-firebird/issues/283
and tried DEFAULT_ENCODING = 'KSC_5601', 'KSC5601' and 'MS949'
but none of them worked.

Can anyone help me?

**Comment by FabianoCastro** [comment]
Hi

From what I understand, there are two configurations, one for functions outside the connection and another for functions inside the connections

![image](https://github.com/hgourvest/node-firebird/assets/34251113/8f16ac95-9fe0-4baf-9b5b-7668f5360248)

**Comment by augustosnk12** [comment]
Hi, 

Even setting these configs I'm still not able to get the correct char encoding

**Comment by augustosnk12** [comment]
@FabianoCastro yes, I did. For example I have in database the field valued as `Produto suíno` and as result I get `Produto su%C3%ADno`
